pkgs/development/python-modules/nose-timer/default.nix

```diff
@@ -1,20 +1,42 @@
-{ buildPythonPackage, fetchPypi, lib, nose, }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, nose
+, mock
+, parameterized
+, termcolor
+}:
 
 buildPythonPackage rec {
   pname = "nose-timer";
   version = "1.0.1";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-j3DRA7f/2RIqWJ3g350Den2WdRm/beEiYh0hhmCbnjo=";
+  src = fetchFromGitHub {
+    owner = "mahmoudimus";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0xsai2l5i1av62y9y0q63wy2zk27klmf2jizgghhxg2y8nfa8x3x";
   };
 
   propagatedBuildInputs = [ nose ];
 
+  checkInputs = [
+    mock
+    nose
+    parameterized
+    termcolor
+  ];
+
+  checkPhase = ''
+    nosetests --verbosity 2 tests
```

Comment:
hooks

Reply:
It was requested to not use the hook in a previous comment here.

Reply:
> It was requested to not use the hook in a previous comment here.

I meant that hooks are missing from the phase, see:

https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/explicit-phases.md